src/components/molecules/MoleculeArticleCard.vue

```diff
@@ -52,6 +54,11 @@ defineProps({
     type: Object as PropType<IExtendedArticle>,
     required: true,
   },
+  displayIcons: {
+    type: Boolean,
+    required: true,
+  },
```

Comment:
As this is a Boolean consider setting a default value and remove the required attribute.
```suggestion
    type: Boolean,
    required: false,
    default: false,
  },
```